test/unit/spec/narrative_core/staticNarrativesManager-spec.js

```diff
@@ -96,16 +96,6 @@ define([
             .andReturn(mockOkResponse(goodServWizResponse));
     }
 
-    function mockErrorServiceWizard() {
```

Comment:
This was failing constantly, is hard to mock with how the clients are written, and the failure case is captured elsewhere already.